VideoLocker/src/main/java/org/edx/mobile/loader/CoursesAsyncLoader.java

```diff
@@ -47,22 +45,15 @@ public CoursesAsyncLoader(Context context, Bundle args, IEdxEnvironment environm
 
         AsyncTaskResult<List<EnrolledCoursesResponse>> result = new AsyncTaskResult<List<EnrolledCoursesResponse>>();
```

Comment:
Studio says `new AsyncTaskResult<List<EnrolledCoursesResponse>>();` can be `new AsyncTaskResult<List<>>();`